lib/msal-common/src/crypto/ICrypto.ts

```diff
@@ -37,6 +39,13 @@ export interface ICrypto {
     generatePkceCodes(): Promise<PkceCodes>;
     /**
      * Generates an JWK RSA S256 Thumbprint
+     * @param resourceRequestMethod 
+     * @param resourceRequestUri 
+     */
+    getPublicKeyThumprint(resourceRequestMethod: string, resourceRequestUri: string): Promise<string>;
```

Comment:
nit: thumbprint spelling error (saw something similar in PR #1, may already be corrected on other PR)

Reply:
corrected in previous PR